api/nakadi-event-bus-api.yaml

```diff
@@ -1151,6 +1263,36 @@ paths:
 # ################################### #
 
 definitions:
+  Storage:
+      type: object
+      description: |
+        This is a storage backend
+        A real one.
+      properties:
+        id:
+          description: The ID of the storage backend
+          type: string
+        storage_type:
+          description: the type of storage (currently, only Kafka)
+          type: string
+        kafka_configuration:
+          description: configuration settings (currently, only Kafka)
```

Comment:
Probably it's better to say that it is filled up only when backend type is kafka. For kinesis one will have `kinesis_configuration`, as far as I understand.